src/gallia/commands/scan/uds/sa_dump_seeds.py

```diff
@@ -209,6 +219,10 @@ async def main(self, args: Namespace) -> None:
                 # Re-enter session. Checking/logging will be done at the beginning of next iteration
                 await self.ecu.set_session(session)
 
+            if args.sleep > 0:
```

Comment:
So above and then:

```suggestion
            if args.sleep is not None:
```

Reply:
The most elegant solution might be this one: Leave the `default=0` thing and just remove the `if`. I think `await asyncio.sleep(0)` is a noop. Choose your favorite. :)